app/src/androidTest/java/com/github/onlynotesswent/ui/overview/EditNoteTest.kt

```diff
@@ -76,6 +78,22 @@ class EditNoteTest {
     org.mockito.kotlin.verify(navigationActions, never()).navigateTo(Screen.OVERVIEW)
   }
 
+  @Test
+  fun clickAddCommentButton() {
+    composeTestRule.onNodeWithTag("Add Comment Button").performClick()
+    composeTestRule.onNodeWithTag("DeleteCommentButton_0").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("EditCommentTextField_0").assertIsDisplayed()
+  }
+
+  @Test
+  fun clickAddCommentButtonAndDelete() {
```

Comment:
Here, you could also test that once the comment is deleted, it is not stored anymore using mocks. But the test is fine anyways